docs/tutorial-diffexp.rst

```diff
@@ -0,0 +1,319 @@
+.. _tutorial-diffexp:
+
+========================
+Differential expressions
+========================
+
+In this tutorial we will learn how to run a differential expression analysis. We will become familiar
+with the concepts of *sample* and *collection*. We will create a collection and add samples to that
```

Comment:
What about "relations"?

My suggestion:
"...concepts of *sample*, *collection* and *relation*. We will create a collection, add samples to that collection and define relations between samples.